polynote-frontend/polynote/data/value_repr.ts

```diff
@@ -118,3 +118,24 @@ ValueRepr.codec = discriminated(
 );
 
 
+// Order of preference of reprs, after preferred MIME repr (TODO: should be a preference?)
```

Comment:
Hmm, I wonder whether there's any way this can be set by type. For example, it makes sense for a Spark DF to have `StreamingDataRepr` first, but for a `PandasDF` we might want to the HTML Repr to be top priority. Could a `ReprsOf` author possibly set a recommended order?

Reply:
Eventual goal is to make this configurable by the user (rather than using this hardcoded `reprOrder`) – like "I'd rather see the HTML first" or "I'd rather see... something to do with the table stuff first (😅 )". But it might be useful for the `ReprsOf` instance to also express some kind of priority. 

Anyway, we'll have a bunch of different orderings to reconcile. Maybe the user even has different preferences for different types 😬

Reply:
I think it might be pretty common to want to default to different priorities based on type, especially wrt "eager" vs. "lazy" data types (e.g., in my example with Spark vs. Pandas DFs.)

Reply:
Agreed, it's something we should play around with in the future.

Reply:
👍